### ISSUE TITLE
manual/javaGuide/main/sql/JavaDatabase.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#567]
- https://github.com/garbagetown/playdocja/blob/2.2.0/documentation/2.2.0/manual/javaGuide/main/sql/JavaDatabase.md

```
--- /Users/garbagetown/Desktop/2.1.5/manual/javaGuide/main/sql/JavaDatabase.md  2013-09-19 22:53:02.000000000 +0900
+++ /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/sql/JavaDatabase.md  2013-09-19 10:11:22.000000000 +0900
@@ -2,9 +2,17 @@

 ## Configuring JDBC connection pools

-Play 2.0 provides a plugin for managing JDBC connection pools. You can configure as many databases you need.
+Play provides a plugin for managing JDBC connection pools. You can configure as many databases you need.

-To enable the database plugin, configure a connection pool in the `conf/application.conf` file. By convention the default JDBC data source must be called `default`:
+To enable the database plugin add javaJdbc in your build dependencies :
+
+'''scala
+val appDependencies = Seq(
+  javaJdbc
+)
+'''
+
+Then you must configure a connection pool in the `conf/application.conf` file. By convention the default JDBC data source must be called `default`:

 '''properties
 # Default database configuration
@@ -59,7 +67,7 @@

 ## Importing a Database Driver

-Other than for the h2 in-memory database, useful mostly in development mode, Play 2.0 does not provide any database drivers. Consequently, to deploy in production you will have to add your database driver as an application dependency.
+Other than for the h2 in-memory database, useful mostly in development mode, Play does not provide any database drivers. Consequently, to deploy in production you will have to add your database driver as an application dependency.

 For example, if you use MySQL5, you need to add a [[dependency| SBTDependencies]] for the connector:


```

**Comment by garbagetown** [comment]
pull requested at #675

**Comment by wm3** [comment]
merged #675
